Content.Server/Singularity/Components/SingularityComponent.cs

```diff
@@ -0,0 +1,84 @@
+using Content.Shared.Singularity.Components;
+using Content.Server.Singularity.EntitySystems;
+using Robust.Shared.Audio;
+
+namespace Content.Server.Singularity.Components;
+
+/// <summary>
+/// The server-side version of <see cref="SharedSingularityComponent">.
+/// Primarily managed by <see cref="SingularitySystem">.
+/// </summary>
+[RegisterComponent]
+[ComponentReference(typeof(SharedSingularityComponent))]
+public sealed class SingularityComponent : SharedSingularityComponent
+{
+    /// <summary>
+    /// The amount of energy this singularity contains.
+    /// If you want to set this go through <see cref="SingularitySystem.SetEnergy"/>
+    /// </summary>
+    [DataField("energy")]
+    [Access(friends:typeof(SingularitySystem), Other=AccessPermissions.Read, Self=AccessPermissions.Read)]
+    public float Energy = 180f;
+
+    /// <summary>
+    /// The rate at which this singularity loses energy over time.
+    /// </summary>
+    [DataField("energyLoss")]
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float EnergyDrain;
+
+    /// <summary>
+    /// The amount of time between energy level updates.
+    /// </summary>
+    [DataField("updatePeriod")]
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float UpdatePeriod = 1.0f;
+
+    /// <summary>
+    /// The amount of time that has elapsed since the last energy level update.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadOnly)]
+    public float _timeSinceLastUpdate = 0f;
+
+#region Audio
+    /// <summary>
+    /// The sound that this singularity produces by existing.
+    /// </summary>
+    [DataField("ambiantSound")]
```

Comment:
ambian -> ambient

Reply:
Already fixed?

Reply:
Oh. I see the issue. I thought you were saying something was missing a t.

Reply:
Done